stanford_courses.module

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * @file
+ * Stanford_courses.module.
+ */
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Component\Utility\UrlHelper;
+use Drupal\Core\Cache\Cache;
+
+/**
+ * Implements hook_page_attachments().
+ */
+function stanford_courses_page_attachments(array &$attachments) {
+  // Get the route.
+  $route = \Drupal::routeMatch()->getRouteName();
+  if (str_contains($route, 'page_view_courses_list_page_courses_list_page')) {
+    $attachments['#attached']['library'][] = 'stanford_courses/courses_page';
```

Comment:
You could probably just attach to the view's render array instead.

Reply:
done.